packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -360,11 +360,16 @@ describe("locked layer", () => {
         // Act
         // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
         const lines = container.querySelectorAll(".locked-line line");
+        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+        const ray = container.querySelector(".locked-line g");
```

Comment:
This selector seems brittle - could we use a different class for locked rays? E.g. `.locked-ray`?